cmd/createEmail.go

```diff
@@ -0,0 +1,119 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/canonical/identity-platform-admin-ui/internal/authorization"
+	"github.com/canonical/identity-platform-admin-ui/internal/config"
+	"github.com/canonical/identity-platform-admin-ui/internal/kratos"
+	"github.com/canonical/identity-platform-admin-ui/internal/logging"
+	"github.com/canonical/identity-platform-admin-ui/internal/mail"
+	"github.com/canonical/identity-platform-admin-ui/internal/monitoring"
+	"github.com/canonical/identity-platform-admin-ui/internal/openfga"
+	"github.com/canonical/identity-platform-admin-ui/internal/pool"
+	"github.com/canonical/identity-platform-admin-ui/internal/tracing"
+	"github.com/canonical/identity-platform-admin-ui/pkg/identities"
+	"github.com/kelseyhightower/envconfig"
+	"github.com/spf13/cobra"
+)
+
+var createEmailCmd = &cobra.Command{
+	Use:   "create-email [identity-id]",
```

Comment:
`create-email` cmd name is a bit odd. The reason for doing this is for CLI future improvements (these all fall into the `create` subcommand)

```
identity-platform-admin-ui create fga-model
identity-platform-admin-ui create admin
identity-platform-admin-ui create identity
identity-platform-admin-ui create email
```

Also if there are new types of emails coming up in the future, we could add cmd flags to include the cases.

Reply:
I agree the name is odd.
This email is part of the "user invite" flow, so maybe we can change this command name to reflect that? Would that help?
I was thinking some command like:
- `send-invite` or
- `user-invite`

Reply:
gotcha. let me change the name.